addons/account/models/account_account.py

```diff
@@ -409,6 +409,11 @@ def _compute_internal_group(self):
             if account.account_type:
                 account.internal_group = 'off_balance' if account.account_type == 'off_balance' else account.account_type.split('_')[0]
 
+    @api.depends('account_type')
+    def _compute_reconcile(self):
+        for account in self:
+            account.reconcile = account.account_type in ('asset_receivable', 'liability_payable')
```

Comment:
I think it's strange to have both a compute method that depends('account_type') and an onchange('account_type'). Given that the remaining _onchange_account_type() now computes the tax_ids to set on the account, it's probably a good idea to change that as well. :thinking: 

not really a problem, though

Reply:
I think the onchange with `tax_ids` should remain the same, at least for this task, if it's not a problem? :slightly_smiling_face: